package-lock.json

```diff
@@ -1,1266 +1,8 @@
 {
   "name": "msvc-dev-cmd",
   "version": "1.1.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Hi. You downgraded the version of lock file from 2 to 1. Is it on purpose?

Reply:
Nope, that's accidental. Thanks for catching this 👍

I've missed this as I'm typically “let npm whatever the fuck it wants in package-lock.json then git add it”. Should have noticed that it's –1k lines of code.

Now, that said, I can't get my npm use the v2 of the lock file, running it on the same machine that I used before. This tool is incomprehensible.

Reply:
> Now, that said, I can't get my npm use the v2 of the lock file, running it on the same machine that I used before. This tool is incomprehensible.

Hmm, I didn't follow.

> npm **WARN** *read-shrinkwrap* This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

You mean this?

Reply:
Yeah, this. I also see this warning with my

```
$ npm --version
6.14.11
```

that I don't really remember downgrading or upgrading recently.

Which is actually weird, considering that v7 is the current stable.

And while writing this comment I have realized *how* that happened. I've accidentally used a terminal with Emscripten environment activated which has the older version of `npm` in it.

Computers are hard 🤷

Reply:
OK.

For me, I got that warning because I'm developing with npm 6 which is shipped with Node.js 14. I just know that npm 7 is the default version in Node.js 15+.